src/DependencyInjection/Compiler/AddRootDirPass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Sets the available Imagine class name in the container.
+ *
+ * @author Martin Auswöger <martin@auswoeger.com>
+ */
+class AddRootDirPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $container->setParameter(
+            'contao.root_dir',
```

Comment:
We need a way to inject `"%contao.root_dir%/%contao.image.target_path%"` into the resizer service. Should I use the `%kernel.root_dir%` instead?